libtransmission/peer-msgs.cc

```diff
@@ -937,7 +936,7 @@ void sendLtepHandshake(tr_peerMsgsImpl* msgs)
     bool const allow_metadata_xfer = msgs->torrent->is_public();
 
     /* decide if we want to advertise pex support */
-    bool const allow_pex = msgs->session->allows_pex() && msgs->torrent->allows_pex();
+    bool const allow_pex = msgs->torrent->allows_pex();
```

Comment:
good catch removing that redundant session check, since torrent does a session check too :+1: